src/utils.js

```diff
@@ -141,6 +141,31 @@ export const reactHTMLTags = [
 
 
 /**
+ * List of "void element" tags
+ * lifted from React source
+ * https://github.com/facebook/react/blob/6a659606415d4853026df6e6d525274fdc7d35ea/src/renderers/dom/shared/ReactDOMComponent.js#L443
+ */
+export const voidElementTags = {
```

Comment:
yeah - no need to overcomplicate it. especially if we're essentially pulling from React source